setup.py

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env python3
+import os
+import sys
+
 from setuptools import setup
 
 with open("pennylane_rigetti/_version.py") as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
-with open("./requirements.txt") as f:
-    requirements = [req.strip() for req in f.readlines()]
+
+requirements = ["pyquil>=3.0.0,<4.0.0", "qcs-api-client>=0.20.13<0.22.0", "pennylane>=0.18"]
```

Comment:
This reinstates the problem the PR was trying to fix and ignores `requirements.txt`.

Reply:
Hi @genos! Thanks for noticing this. I must've missed it when fixing the merge conflict.

Reply:
Reverted the change. 👍